### DISCUSSION TITLE
Can I download a specifical file from my repo?

### DISCUSSION THREAD

**Discussion by songyuc** [PyGithub/PyGithub#2368] [Q&A]
Hi, guys,
I want to maintain a C++ template file in a repo, and I may need to download it in another project.
So, can I download a specifical file from my repo with PyGithub

Your answer and guide will be appreciated!

**Comment by Vuizur** [comment]
You don't need to use PyGithub for this, all files in a repository are accessible with a static link. You only need to click on the file and then on the "raw" button. Then in the browser bar there you get the link to the file. In Python you can then download it using standard file download methods like detailed on Stackoverflow here: https://stackoverflow.com/questions/22676/how-to-download-a-file-over-http
